Lanchonete/Lanchonete/ Features/Menu/Presentation/TableDemoViewPresenter.swift

```diff
@@ -26,40 +25,16 @@ public class TableDemoPresenter {
         return menuManager.getSandwichTypes()
     }()
     
-    //Get all Ingredients from the menu
-    // Get all the sandwiches from the menu
-    lazy var ingredients: [Ingredient] = {
-        return ingredientManager.getIngredientList()
-    }()
-    
     func numberOfSections() -> Int {
         return Sections.allCases.count
     }
     
-    func titleForSection(_ section: Int) -> String {
-        let section = Sections.allCases[section]
-        
-        switch section {
-        case .menuSandwiches:
-            return "Menu Sandwiches"
-        case .ingredients:
-            return "Available Ingredients to personalize"
-        }
+    func titleForSection() -> String {
+        return "Menu Sandwiches"
     }
     
-    func numberOfRowsInSection(_ section: Int) -> Int {
-        let section = Sections.allCases[section]
-        
-        switch section {
-        case .menuSandwiches:
-            return menuSandwiches.count
-        case .ingredients:
-            return ingredients.count
-        }
-    }
-    
-    func getSectionHeaders() -> [Sections]{
-        return Sections.allCases
+    func numberOfRowsInSection() -> Int {
+        return menuSandwiches.count
```

Comment:
`TableDemoViewPresenterTests` is quite tiny, no tests validate the expected rows and sections of the menu 😛

Reply:
Hey Carlos, that's true, can I do the tests in other occasion?